pkgs/development/python-modules/swh-scanner/default.nix

```diff
@@ -36,15 +36,6 @@ buildPythonPackage rec {
     hash = "sha256-baUUuYFapBD7iuDaDP8CSR9f4glVZcS5qBpZddVf7z8=";
   };
 
-  patches = [
-    # To be removed at the next release
-    # See https://gitlab.softwareheritage.org/swh/devel/swh-scanner/-/merge_requests/160
-    (fetchpatch {
-      url = "https://gitlab.softwareheritage.org/swh/devel/swh-scanner/-/commit/0eb273475826b0074844c7619b767c052562cfe4.patch";
```

Comment:
That patch is in upstream's main branch but hasn't made it to any release yet. The only reason we can remove it is that the tests are now disabled.